{"tests": [{"data": {"status": "failure", "tool_id": "mdtraj_traj_converter", "time_seconds": 0.40175294876098633, "inputs": {"input|input_format": "xtc", "output_format": "dcd", "input|input_file": {"src": "hda", "id": "2891970512fa2d5a"}}, "tool_version": "1.9.1", "problem_log": "Traceback (most recent call last):\n  File \"/usr/lib/python2.7/unittest/case.py\", line 329, in run\n    testMethod()\n  File \"/tmp/tmpZbCYDd/galaxy-dev/test/functional/test_toolbox.py\", line 88, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpZbCYDd/galaxy-dev/test/functional/test_toolbox.py\", line 41, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpZbCYDd/galaxy-dev/lib/galaxy/tools/verify/interactor.py\", line 682, in verify_tool\n    assert data_list or data_collection_list\nAssertionError\n", "test_index": 0, "problem_type": "exceptions.AssertionError"}, "id": "functional.test_toolbox.TestForTool_mdtraj_traj_converter.test_tool_000000", "has_data": true}], "version": "0.1", "exit_code": 1, "summary": {"num_skips": 0, "num_errors": 0, "num_failures": 1, "num_tests": 1}}